lib/openhab/dsl/items/timed_command.rb

```diff
@@ -30,6 +30,8 @@ module TimedCommand
         #
         # @attr [Item] item
         #   @!visibility private
+        # @attr [Object] command
```

Comment:
it's probably best to label this as accepting a Core::Types::Command, not an Object. technically, yes, it can be an object, but that's rarely helpful in documentation. then we should probably add docs to Command (and State, for GenericItem#update) that most places a Command (or Type) is accepted, types that are convertible to the proper type for the applicable item are also accepted.

Reply:
+1